fares-data-build-tool/fdbt-site
# feat: CFD-245 Reusing groups

# Description

-   Reuse saved groups on the passenger types screen

# Testing instructions

-   Go through the single ticket journey, selecting to define a new group
-   Set all the group options and choose to name your new group when given the chance
-   Continue until you end up on the time restrictions page
-   Go back and start a new journey
-   This time select to reuse a saved group
-   Check you are taken direct to the time restrictions page
-   Complete the journey and check that the previous selected group configuration is used

# Type of change


# Checklist:
